bottery/cli.py

```diff
@@ -49,4 +54,5 @@ def startproject(name):
 @debug_option
 def run(port, debug):
     app = App()
-    app.run()
+    app.run
+    
```

Comment:
Why you changed this lines? One of the errors you got come from the empty line at the end of this file.